renovate.json

```diff
@@ -1,10 +1,4 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base",
-    ":enablePreCommit",
```

Comment:
Seems to be missing preCommit

Reply:
This is now part of the default preset: https://github.com/hetznercloud/.github/pull/11